src/database/volatile/mod.rs

```diff
@@ -55,3 +67,68 @@ impl Tabular for Database<'_> {
         todo!()
     }
 }
+
+#[cfg(test)]
+mod tests {
+
+    use super::*;
+
+    /// A maximally simple Record implementation, consisting of only a BitVec value
```

Comment:
"Maximally simple" lol

These days we have actual record implementations, which should help here.